clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -0,0 +1,233 @@
+                //       {labels.primaryButton}
+                //     </button>
+                //   </div>
+                // </div>
+                cy.get("div#fides-consent-banner.fides-consent-banner").within(() => {
+                    cy.get("div#fides-consent-banner-description.fides-consent-banner-description")
+                        .contains("This test website is overriding the banner description label.");
+                    cy.get("div#fides-consent-banner-buttons.fides-consent-banner-buttons").within(() => {
+                        cy.get("button#fides-consent-banner-button-tertiary.fides-consent-banner-button.fides-consent-banner-button-tertiary")
+                            .contains("Customize Test");
+                        cy.get("button#fides-consent-banner-button-secondary.fides-consent-banner-button.fides-consent-banner-button-secondary")
+                            .contains("Reject Test");
+                        cy.get("button#fides-consent-banner-button-primary.fides-consent-banner-button.fides-consent-banner-button-primary")
+                            .contains("Accept Test");
+
+                        // Order matters - it should always be tertiary, then secondary, then primary!
+                        cy.get("button").within(buttons => {
+                            const buttonIds = buttons.map((_, elem) => elem.id).get();
+                            cy.wrap(buttonIds).should("have.ordered.members", [
+                                "fides-consent-banner-button-tertiary",
+                                "fides-consent-banner-button-secondary",
+                                "fides-consent-banner-button-primary",
+                            ]);
+                        });
+                    })
+                });
+            });
+
+            it("should allow accepting all", () => {
+                cy.contains("button", "Accept Test").should("be.visible").click();
+                // TODO: consent.cy.ts uses cy.waitUntil to wait longer for the cookie to save in CI - will this be needed?
+                cy.getCookie(CONSENT_COOKIE_NAME).should("exist").then((cookie) => {
+                    const cookieKeyConsent = JSON.parse(
+                        decodeURIComponent(cookie!.value)
+                    );
+                    expect(cookieKeyConsent).property("data_sales").is.eql(true);
+                    expect(cookieKeyConsent).property("tracking").is.eql(true);
+                })
+                cy.contains("button", "Accept Test").should("not.be.visible");
+            });
+
+            it("should support rejecting all consent options", () => {
+                cy.contains("button", "Reject Test").should("be.visible").click();
+                // TODO: consent.cy.ts uses cy.waitUntil to wait longer for the cookie to save in CI - will this be needed?
+                cy.getCookie(CONSENT_COOKIE_NAME).should("exist").then((cookie) => {
```

Comment:
It wil be very much needed! I lost like an hour of sanity fighting with cookies in CI. There's a helper now called `cy.waitUntilCookieExists(...)` that you should use, see examples in consent.cy.ts